packages/react/src/dialog/popup/useDialogPopup.tsx

```diff
@@ -51,13 +46,6 @@ export function useDialogPopup(parameters: useDialogPopup.Parameters): useDialog
     return initialFocus;
   }, [defaultInitialFocus, initialFocus, openMethod]);
 
-  useEnhancedEffect(() => {
-    setPopupElementId(id);
-    return () => {
-      setPopupElementId(undefined);
-    };
-  }, [id, setPopupElementId]);
-
```

Comment:
Reverting this to match other popups, as I added `useRole`.

Issue to make them all handle `id` correctly: #1552